### ISSUE TITLE
View Lineage Error for AzureSQL/MSSQL

### ISSUE THREAD

**Issue by ulixius9** [open-metadata/OpenMetadata#8424]
**Affected module**
Ingestion Framework

**Describe the bug**
View lineage for AzureSQL or MSSQL does not get generated when the view definition contains `[]`

so the lineage will be generated for the view with definition like this 
```
create view sales.new_product_view as select * from production.products;
```

but view lineage will not be generated for queries like 
```
create view [sales].[product_view] as select * from [production].[products];
```

**Comment by eeshan-deshpande** [comment]
This is happening for Table level lineage also.

**Comment by harshach** [comment]
@nahuelverdugo @ulixius9 lets take it up for 0.13.1

**Comment by nahuelverdugo** [comment]
@harshach addressed in #9257
